models/Profile.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
```

Comment:
did you forget to delete this file?

Reply:
Could have sworn all profile files had been removed but I must have reverted that accidentally. removed and can be ignored